pkg/i18n/english.go

```diff
@@ -1271,7 +1280,9 @@ func EnglishTranslationSet() *TranslationSet {
 		InteractiveRebaseTooltip:             "Begin an interactive rebase with a break at the start, so you can update the TODO commits before continuing.",
 		RebaseOntoBaseBranchTooltip:          "Rebase the checked out branch onto its base branch (i.e. the closest main branch).",
 		MustSelectTodoCommits:                "When rebasing, this action only works on a selection of TODO commits.",
-		ConfirmMerge:                         "Are you sure you want to merge '{{.selectedBranch}}' into '{{.checkedOutBranch}}'?",
+		SquashMergeUncommitted:               "Squash merge '{{.selectedBranch}}' into the current working tree.",
```

Comment:
"current working tree" reads a bit strange, as if there were several. I'd probably just drop the word "current".